addons/account/models/account_payment_method.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields
+
+
+class AccountPaymentMethod(models.Model):
+    _name = "account.payment.method"
+    _description = "Payment Methods"
+    _order = 'sequence'
+
+    name = fields.Char(required=True, translate=True)
```

Comment:
Could you add `active=fields.Boolean(default=True)` to hide unused method during the configuration of journal.

Reply:
No! All this stuff is about to change. See: https://github.com/odoo/odoo/pull/67331